src/wallet/wallet.cpp

```diff
@@ -1619,6 +1619,8 @@ int64_t CWallet::RescanFromTime(int64_t startTime, const WalletRescanReserver& r
  * @param[in] stop_block  Scan ending block. If block is not on the active
  *                        chain, the scan will continue until it reaches the
  *                        chain tip.
```

Comment:
Is this comment still relevant? And the mention of `stop_block` in `@pre` comment?